contributors/guide/first-contribution.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "Making your First Contribution"
+weight: 2
+slug: "first-contribution" 
+---
+
+
+- [Your First Contribution](#your-first-contribution)
+  - [Find something to work on](#find-something-to-work-on)
+    - [Find a good first topic](#find-a-good-first-topic)
+      - [Issue Assignment in Github](#issue-assignment-in-github)
+    - [Learn about SIGs](#learn-about-sigs)
+      - [SIG structure](#sig-structure)
+      - [Find a SIG that is related to your contribution](#find-a-sig-that-is-related-to-your-contribution)
+    - [SIG-specific contributing guidelines](#sig-specific-contributing-guidelines)
+    - [File an Issue](#file-an-issue)
+
+# Your First Contribution
```

Comment:
Same as with the contributing, I'd move the below text above the ToC and remove the top line from the ToC itself which should remove the extra indent.